component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAuthenticator.java

```diff
@@ -493,6 +555,26 @@ private void validateAccountLockStatusForLocalUser(AuthenticationContext context
         }
     }
 
+    private void setErrorContextWhenAccountLocked(String username) throws AuthenticationFailedException {
+        String accountLockedReason = "";
```

Comment:
add a new line after the method signature

Reply:
Addressed in 807d553798051918aaacd7d3f11357bb3eceec73